pom.xml

```diff
@@ -1,140 +1,141 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.7.4</version>
-        <relativePath/>
-    </parent>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This should have been done during checkstyle task, shouldn't it? I don't expect to see such changes in "logout" branch/PR. Do I understand correctly that pom.xml was not formatted properly and you have only changed formatting of the file? I want to be 100% sure that no dependencies were added/removed here.

Reply:
changed to previous code